### PR TITLE
Separate subnet_ids and subnet_names in boto_vpc.route_table_present

### PR DESCRIPTION
**Pull request by mgwilliams**
@claudiupopescu: per our conversation

### CODE REVIEW

File: salt/modules/boto_vpc.py [hunk 1]
```diff
@@ -837,10 +837,10 @@ def subnet_exists(subnet_id=None, name=None, subnet_name=None, cidr=None,
         subnets = conn.get_all_subnets(**filter_parameters)
         log.debug('The filters criteria {0} matched the following subnets:{1}'.format(filter_parameters, subnets))
         if subnets:
-            log.info('Subnet {0} exists.'.format(subnet_id))
+            log.info('Subnet {0} exists.'.format(subnet_name or subnet_id))
```
Comment (jfindlay) [inline salt/modules/boto_vpc.py]: This will produce `Subnet True exists`.  Did you mean `subnet_name if subnet_name else subnet_id`?

Comment (mgwilliams) [inline salt/modules/boto_vpc.py]: @jfindlay I don't think that's true...

```
>>> 'alpha' or 'beta'
'alpha'
>>> None or 'beta'
'beta'
```

Comment (jfindlay) [inline salt/modules/boto_vpc.py]: Interesting, I guess I'm wrong, sorry.

**Comment by mgwilliams** [comment]
@jfindlay thanks for merging!

**Pull request merged**